src/lib/utils/thorchain/memo/assertAndProcessMemo.ts

```diff
@@ -82,13 +82,33 @@ export const assertAndProcessMemo = (memo: string): string => {
     case '=':
     case 's': {
       // SWAP:ASSET:DESTADDR:LIM/INTERVAL/QUANTITY:AFFILIATE:FEE
-      const [_action, asset, destAddr, limit, , fee] = memo.split(':')
+      const [
+        _action,
+        asset,
+        destAddr,
+        limit,
+        ,
+        fee,
+        aggregatorLastTwoChars,
+        finalAssetAddressShortened,
+        finalAssetAmountOut,
```

Comment:
To honor THOR terminology, and not to introduce stripping ramifications in this function, which it shouldn't be concerned about.

```suggestion
        aggregatorAddress,
        finalAssetAddress,
        finalAssetAmountOut,
```

<img width="827" alt="image" src="https://github.com/shapeshift/web/assets/17035424/3cdbf32d-579d-4264-9704-3b1ecbc7f37f">

Reply:
Done in 03978b18e16